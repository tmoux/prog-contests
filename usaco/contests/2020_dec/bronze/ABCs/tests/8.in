408165664 132722948 275442716 144935563 263230101 130507153 277658511
