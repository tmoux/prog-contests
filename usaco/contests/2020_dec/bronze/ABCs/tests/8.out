130507153 132722948 144935563
